### ISSUE TITLE
WindowsError on dscheduler

### ISSUE THREAD

**Issue by philastrophist** [dask/distributed#269]
So it appears that Bokeh is not found, but I have Bokeh and everything is up to date.

Interesting...

here lieth the traceback:

```
distributed.scheduler - INFO - Start Scheduler at:          192.168.0.7:8786
distributed.scheduler - INFO -            http at:          192.168.0.7:9786
distributed.scheduler - WARNING - Could not start Bokeh web UI
Traceback (most recent call last):
  File "C:\Anaconda\lib\site-packages\distributed\cli\dscheduler.py", line 88, in main
    bokeh_proc = subprocess.Popen(args)
  File "C:\Anaconda\lib\subprocess.py", line 710, in __init__
    errread, errwrite)
  File "C:\Anaconda\lib\subprocess.py", line 958, in _execute_child
    startupinfo)
WindowsError: [Error 2] The system cannot find the file specified
Traceback (most recent call last):
  File "C:\Anaconda\Scripts\dscheduler-script.py", line 5, in <module>
    sys.exit(distributed.cli.dscheduler.go())
  File "C:\Anaconda\lib\site-packages\distributed\cli\dscheduler.py", line 107, in go
    main()
  File "C:\Anaconda\lib\site-packages\click\core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "C:\Anaconda\lib\site-packages\click\core.py", line 696, in main
    rv = self.invoke(ctx)
  File "C:\Anaconda\lib\site-packages\click\core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Anaconda\lib\site-packages\click\core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "C:\Anaconda\lib\site-packages\distributed\cli\dscheduler.py", line 100, in main
    bokeh_proc.terminate()
UnboundLocalError: local variable 'bokeh_proc' referenced before assignment
```

**Comment by mrocklin** [comment]
Indeed.  That looks quite grim.  Resolving now...

**Comment by mrocklin** [comment]
Should be resolved in #270 

I've tested on a local VM and everything checks out.  It's a bit concerning that tests didn't pick this up
